src/index.js

```diff
@@ -44,8 +44,8 @@ async function run() {
   core.info('Fetching changed files in the pull request');
   const changed_files = await github.fetch_changed_files();
 
-  core.info('Fetching currently requested reviewers');
-  const current_reviewers = await github.fetch_current_reviewers();
+  core.info('Fetching previously requested reviewers');
+  const current_reviewers = await github.fetch_reviewers();
   core.info(`Already in review: ${current_reviewers.join(', ')}`);
```

Comment:
Should `current_reviewers` be renamed, since the log now talks about the previous reviewers?